store/conf/attrs/zimbra-attrs.xml

```diff
@@ -8461,7 +8461,7 @@ TODO: delete them permanently from here
   <desc>whether or not an account represents a Mobile Gateway app</desc>
 </attr>
 
-<attr id="1761" name="zimbraDataSourceOAuthToken" type="string" cardinality="single" optionalIn="imapDataSource" since="8.7.0">
+<attr id="1761" name="zimbraDataSourceOAuthToken" type="string" cardinality="single" optionalIn="dataSource,imapDataSource" flags="ephemeral" since="8.7.0">
```

Comment:
We will need a lot of testing and probably changes in this area when we add new ephemeral attributes.
Suspect it will work fine when first switching on using an ephemeral store but if that is already in use, adding new ones may/will probably lose all values stored in LDAP for that new attribute (that may not be a big issue in this case but it needs thinking about)

Reply:
I understand. It seems the attribute is not used in zm-mailbox now, but it will be better to  create a new attribute for NIC project. Thanks.